internal/store/postgres/servicedata.go

```diff
@@ -19,6 +19,11 @@ type Key struct {
 	DeletedAt   sql.NullTime `db:"deleted_at"`
 }
 
+type UpsertServiceData struct {
+	Key   string `db:"key.key"`
+	Value string `db:"data.value"`
+}
```

Comment:
The model here should ideally have the same fields as the DB columns
and should not be having complex nested field separated with `.` like this